bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -74,6 +74,53 @@ spec:
                           description: dnsName is the domain name to allow/deny traffic to. If this is set, cidrSelector must be unset.
                           pattern: ^([A-Za-z0-9-]+\.)*[A-Za-z0-9-]+\.?$
                           type: string
+                        nodeSelector:
+                          description: nodeSelector will allow/deny traffic to the
```

Comment:
I compared the changes with upstream ones that landed, the `cidrSelector` and `dnsName` descriptions are not updated in this one.